Mathlib/Algebra/Equiv/TransferInstance.lean

```diff
@@ -600,6 +600,35 @@ noncomputable def _root_.Shrink.linearEquiv [Small.{v} α] [AddCommMonoid α] [M
     Shrink.{v} α ≃ₗ[R] α :=
   Equiv.linearEquiv _ (equivShrink α).symm
 
+variable {R : Type*} [CommSemiring R]
```

Comment:
I am confused by this. We already have `variable {R : Type*}` in scope (line 529). Is it really wise to make a new `R`?

Reply:
I am not so worried, since indeed this is the end of the section anyway. Nevertheless, I moved it into a different section.